manage.py

```diff
@@ -473,6 +510,82 @@ def correct_user_permissions():
     db.commit()
 
 
+def _create_field_list_field(summary_table, tables_only):
```

Comment:
```suggestion
def _add_field_list_column(summary_table, tables_only):
```

Change method call as well.